helix-loader/src/config.rs

```diff
@@ -19,7 +19,7 @@ pub fn user_lang_config() -> Result<toml::Value, toml::de::Error> {
         .into_iter()
         .chain([default_lang_config()].into_iter())
         .fold(toml::Value::Table(toml::value::Table::default()), |a, b| {
-            crate::merge_toml_values(b, a)
+            crate::merge_toml_values(b, a, false)
```

Comment:
Before #2129 we weren't merging `config.toml` anywhere. Now that #2129 has been merged we can disable the array merging by passing in that flag as `false`.

Reply:
I think you linked the wrong PR.

Reply:
Whoops! 🤦 

I meant https://github.com/helix-editor/helix/pull/1249